### DISCUSSION TITLE
What's the difference between `kitty +kitten` and `kitten`?

### DISCUSSION THREAD

**Discussion by SPiCaRiA** [kovidgoyal/kitty#7212] [Q&A] [answered]
Hi, sorry if this is a dumb one but I'm really curious but couldn't find an answer after going over the man pages and the official website.

 I'm confused about the difference between `kitty +kitten` and `kitten`:
1. I feel that they work the same for some of the kittens (e.g. themes, ask, etc.)
2. The completion for each is slightly different (e.g.`kitten` supports `kitten @` but `kitty +kitten` doesn't)

Also, is `kitten @` different from `kitty @`?

Thanks!

**Comment by kovidgoyal** [comment, marked as answer]
kitty +kitten should be used to run your own kittens written in python.

kitten is all the built-in kittens bundled in a single statically
compiled executable and should be used when invoking any builtin
kittens.

kitty @ and kitten @ are the same, the former is present only for
backwards compat and should not be used.
